### ISSUE TITLE
Which BNO055 USB Stick version do you use?

### ISSUE THREAD

**Issue by BillWSY** [yoshito-n-students/bno055_usb_stick#6]
Hi,

I got my BNO055 USB stick from Digikey today (https://www.digikey.com/product-detail/en/bosch-sensortec/BNO055-USB-STICK/828-1045-ND/6136288). But when I plug it in I didn't see /dev/ttyACM0 showing up. So I was wondering if you are using the same hardware (Red PCB) or the green PCB version.

Thanks in advance.

**Comment by yoshito-okada** [comment]
I am using Red PCBs with Ubuntu 16.04 and ROS kinetic.  I guess no tty device means a  problem between your OS and the PCB.

**Comment by BillWSY** [comment]
Yes -- I guess so. Did you upgrade the firmware on the USB stick?

I can't even get a serial port under windows. When you used Windows to get the protocol, which Windows version did you use?

Thank you in advance.

**Comment by yoshito-okada** [comment]
Firmware upgrade of the USB stick: No
When I got the protocol: Nov 2016 (just before the first commit of this repo)
Version of windows I got the protocol on: Windows 10

**Comment by BillWSY** [comment]
Thank you very much for the information!

**Comment by BillWSY** [comment]
Some update about my USB stick:

I purchased my BNO055 USB stick from Digikey. They didn't work in Linux at the beginning (dmesg shows `cdc_acm: probe of ... failed with error -22`, and `/dev/ttyACM0` didn't exist). I tried to plug the USB stick into a Windows 10 computer, but it wasn't recognized as a COM device, and Bosch Development Desktop didn't show a COM port in either application mode and bootloader mode.

I ended up using a Windows 7 computer to update the firmware. After updating the firmware, the USB stick works under Ubuntu without any issue.

Again, thank you, Yoshito! Your information and experience were really helpful to me. Much appreciated!
